src/test/java/com/infobip/jira/CommitMessageVersionExtractorTest.java

```diff
@@ -48,7 +49,7 @@ public void shouldSuccessfullyExtractVersionFromReleaseMessage() {
 
         whenExtractVersion("Release 1.0.0");
 
-        then(version).isEqualTo(Optional.of("1.0.0"));
+        then(version).contains("1.0.0");
```

Comment:
ah, not string contains text but Optional contains content (more correctly, is present) ... ugh, language fail ... you can add `.isPresent()` as a reminder that this is `Optional`, but that's superfluous